### PR TITLE
Add timeseries chart

### PR DESCRIPTION
**Pull request by balajimanoharan**
Screen Shot 2020-05-07 at 10 40 38 AM

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/chart-container.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          month = d.getMonth(),
+          date = d.getDate();
+
+    month = month > 10 ? month : `0${month}`;
+    date = date > 10 ? date : `0${date}`;
+    return `${year}-${month}-${date}T00:00Z`;
+  }
+
+  @(task(function* (location, lookbackDate) {
+    const records = yield this.elide.fetch.perform('healthRecords', {
```
Comment (jkusa) [inline app/components/chart-container.js]:
```suggestion
    const records = yield this.elide.fetch.linked().perform('healthRecords', {
```

File: app/components/chart-container.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  @tracked modalChart;
+
+  //feature flag bar chart
+  showBarChart = false;
+
+  @action
+  fetchData() {
+    const { location } = this.args;
+    if (location) {
+      this.fetchRecords.perform(location, this.lookbackDate);
```
Comment (kevinhinterlong) [inline app/components/chart-container.js:30]: Clear out old results before loading new ones
```suggestion
      this.records = undefined;
      this.fetchRecords.perform(location, this.lookbackDate);
```

**Review by kevinhinterlong — COMMENTED** [review]
Would be nice to deduplicate the charts and intl the series names

File: app/components/chart-container.hbs [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+            width="10px"
+            height="10px"
+          }}
+        </span>
+      {{/unless}}
+    </div>
+  {{/if}}
+
+  {{#if this.showModal}}
+    <div class="chart-container__modal modal is-active" ...attributes>
```
Comment (kevinhinterlong) [inline app/components/chart-container.hbs:28]: Kind of annoying that you have to press the `x` to close the modal

Comment (balajimanoharan) [reply app/components/chart-container.hbs:28]: thats how the denali modal works.

File: app/components/chart-container.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    }
+  }
+
+  get lookbackDate() {
+    const d = new Date();
```
Comment (jkusa) [inline app/components/chart-container.js]: https://date-fns.org/

File: app/components/stacked-bar.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+ * Licensed under the Apache License, Version 2.0. See accompanying LICENSE file for terms.
+ */
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import numeral from 'numeral';
+
+const METRICS = {
+  'numActiveCases': 'Active',
+  'numDeaths': 'Fatal',
+  'numRecoveredCases': 'Recovered'
```
Comment (jkusa) [inline app/components/stacked-bar.js]: intl

File: app/components/time-series.hbs [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  <ApexChart
+    class="timeseries__chart"
+    @type="line"
+    @width={{@width}}
+    @height={{@height}}
+    @series={{this.series}}
+    @chartOptions={{this.chartOptions}}
+  />
+</div>
+
```
Comment (jkusa) [inline app/components/time-series.hbs]:
```suggestion
```

File: app/components/time-series.js [hunk 7]
```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright 2020, Verizon Media.
+ * Licensed under the Apache License, Version 2.0. See accompanying LICENSE file for terms.
+ */
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import numeral from 'numeral';
+
+const METRICS = {
+  'totalConfirmedCases': 'Confirmed',
```
Comment (jkusa) [inline app/components/time-series.js]: intl

File: ember-cli-build.js [hunk 8]
```diff
@@ -30,6 +30,13 @@ module.exports = function (defaults) {
     'ember-fetch': {
       preferNative: true,
     },
+    svgJar: {
+      optimizer : {
+        plugins: [
+          { removeTitle: true }
```
Comment (jkusa) [inline ember-cli-build.js:39]: nice!

File: tests/unit/services/elide-test.js [hunk 9]
```diff
@@ -54,7 +54,22 @@ module('Unit | Service | elide', function (hooks) {
...
+    );
+  });
+
+  test('_buildFetchUrl - gt, lt, ge & le', function (assert) {
+    const { service } = this;
+    const actualUrl = service._buildUrl('healthRecords', {
+      gt: { referenceDate: '2020-04-03T00:00Z' },
+      ge: { referenceDate: '2020-04-04T00:00Z' },
+      lt: { referenceDate: '2020-05-01T00:00Z' },
+      l3: { referenceDate: '2020-04-30T00:00Z' }
```
Comment (jkusa) [inline tests/unit/services/elide-test.js]: `le`?

File: app/components/chart-container.js [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+ * Licensed under the Apache License, Version 2.0. See accompanying LICENSE file for terms.
+ */
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { task } from 'ember-concurrency';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { format, subDays } from 'date-fns';
+
+const DATA_LOOKBACK = 30;
```
Comment (jkusa) [inline app/components/chart-container.js]:
```suggestion
const DATA_LOOKBACK = 90;
```

**Pull request merged**